docs/readthedocs/source/doc/LLM/Quickstart/install_windows_gpu.md

```diff
@@ -2,25 +2,18 @@
 
 This guide demonstrates how to install IPEX-LLM on Windows with Intel GPUs. 
 
-It applies to Intel Core Ultra and Core 12 - 14 gen integrated GPUs (iGPUs), as well as Intel Arc Series GPU.
+It applies to Intel Core Ultra and Core 11 - 14 gen integrated GPUs (iGPUs), as well as Intel Arc Series GPU.
```

Comment:
Changed to 11 as we currently done some verification on i7-1185G7
https://github.com/analytics-zoo/nano/issues/1337#issuecomment-2105443131